tests/Microsoft.Bot.Builder.Azure.Tests/CosmosDbStorageTests.cs

```diff
@@ -407,6 +411,76 @@ public async Task WaterfallCosmos()
             }
         }
 
+        [TestMethod]
+        public async Task DeleteAsyncFromSingleCollection()
+        {
+            string documentId = "UtteranceLog-001";
+            string[] keys = { documentId };
+            var storage = new CosmosDbStorage(CreateCosmosDbStorageOptions());
+            var item = new StoreItem { MessageList = new string[] { "hi", "how are u" }, City = "Contoso" };
+            var changes = new Dictionary<string, object>();
+            {
```

Comment:
I don't understand the presence of the `{` & `}` for adding the `StoreItem` in the `changes` Dictionary. Shouldn't it be this instead:
```
var changes = new Dictionary<string, object>();
changes.Add(documentId, item);
```